### PR TITLE
New Always Fails Plutus script

### PR DESCRIPTION
**Pull request by newhoggy**

### CODE REVIEW

File: plutus-example/plutus-example/src/Cardano/PlutusExample/AlwaysFails.hs [hunk 1]
```diff
@@ -0,0 +1,42 @@
+{-# LANGUAGE DataKinds #-}
+{-# LANGUAGE FlexibleContexts #-}
+{-# LANGUAGE NoImplicitPrelude #-}
```
Comment (Jimbo4350) [inline plutus-example/plutus-example/src/Cardano/PlutusExample/AlwaysFails.hs:3]: Do we need all of these constraints?

File: scripts/plutus/scripts/guess-42-datum-42-txin.plutus [hunk 2]
```diff
@@ -0,0 +1,5 @@
+{
+    "type": "PlutusScriptV1",
```
Comment (Jimbo4350) [inline scripts/plutus/scripts/guess-42-datum-42-txin.plutus:2]: Should this be called always-succeeds?

File: plutus-example/plutus-example/src/Cardano/PlutusExample/AlwaysFails.hs (hunk 1 above)
Comment (newhoggy) [reply plutus-example/plutus-example/src/Cardano/PlutusExample/AlwaysFails.hs:2]: Removed unnecessary language pragmas

File: scripts/plutus/scripts/guess-42-datum-42-txin.plutus (hunk 2 above)
Comment (newhoggy) [reply scripts/plutus/scripts/guess-42-datum-42-txin.plutus]: I've removed the file.

**Review by Jimbo4350 — APPROVED** [review]
LGTM!

**Comment by newhoggy** [comment]
bors merge

**Comment by newhoggy** [comment]
bors merge

**Pull request merged**